lightkurve/lightcurve.py

```diff
@@ -642,6 +645,7 @@ def normalize(self, unit='unscaled'):
             from zero.
         """
         validate_method(unit, ['unscaled', 'percent', 'ppt', 'ppm'])
+        validate_method(method, ['median', 'mean'])
```

Comment:
If we do allow users to pass a function to the `method` keyword argument this will break.